docs/dc-apps-performance-toolkit-user-guide-jira.md

```diff
@@ -38,6 +38,24 @@ You are responsible for the cost of the AWS services used while running this Qui
 
 To reduce costs, we recommend you to keep your deployment up and running only during the performance runs.
 
+### AWS cost estimation ###
+We have made the research on estimation our Jira stack costs. We have used next parameters:
+- EC instance
+- DB instance
+- EFS
+Based on our research results we have next indicators:
+
+_*The prices below are approximate and may vary depending on factors such as (region, scalability, etc.)_
+
+| Instances      | Costs ($/hr)  |
+| ---------      | --------------|          
+| One instance   | ≈ 1 - 1,3     |
```

Comment:
I think we should add one more column where we should add an approximate minimum of hours used by this configuration. For example, one node we need at least for 3 hours, 2 nodes for 2 hours and so on. And then we could also add the total minimum.

Reply:
I have doubts that we can correctly estimate the minimum hours needed. Or it would be frustrating for a vendor that he uses Stack much more time than mentioned in the documentation.

Reply:
But in another case, the cost per hour will not tell the vendor the money he can spend because he has no idea about the approximate time. We can provide the time we spend by our team and emphasize that he will spend more time and cost will be higher and the vendor could predict cost by multiplying it by 2 or 3 for having approximately digit. Also, in the future, we can add this not required question to the checklist and get a statistic of how much vendors spend money on the testing process using AWS and provide this information instead of ours

Reply:
Approximate timings are specified in User Guides. I don't like the idea of putting any numbers here because those numbers may vary a lot depending on too many paraments which we can not control.

Reply:
Okay

Reply:
Yep! I'm afraid that I may not have included all the parameters that we use in our stack, so currently, this estimation very approximate!